app/src/main/java/com/github/multimatum_team/multimatum/activity/SearchLocationActivity.kt

```diff
@@ -5,12 +5,54 @@ import android.os.Bundle
 import android.view.View
 import androidx.appcompat.app.AppCompatActivity
 import com.github.multimatum_team.multimatum.R
+import com.mapbox.search.*
+import com.mapbox.search.result.SearchResult
+import com.mapbox.search.result.SearchSuggestion
 import com.mapbox.search.ui.view.SearchBottomSheetView
 
 
 class SearchLocationActivity : AppCompatActivity() {
 
     private lateinit var searchBottomSheetView: SearchBottomSheetView
+    val searchEngine = MapboxSearchSdk.getSearchEngine()
```

Comment:
Maybe this should be private?

Reply:
Yes, thanks for the review. Very relevant comments and suggestions.